cat_server/helloworld/unikernel.ml

```diff
@@ -3,6 +3,9 @@ open V1_LWT
 
 module Main (C:CONSOLE) = struct
 
-  let start console = OS.Time.sleep 2.0 >>
+  let start console =
+    OS.Time.sleep 2.0
+
```

Comment:
Interesting. In this case (and elsewhere) I use the blank line to indicate
the lwt scheduler activation- I don't find the >>= fun line noise
particularly visible. But, as a long time python user, I do see the
horizontal indentation and the let binding as significantly visible. Guess
it's down to what you're used to reading...
On 7 Apr 2016 1:19 p.m., "Daniel Bünzli" <notifications@github.com> wrote:

> In cat_server/helloworld/unikernel.ml
> <https://github.com/mattgray/devwinter2016/pull/2#discussion_r58861091>:
>
> > @@ -3,6 +3,9 @@ open V1_LWT
> >
> >  module Main (C:CONSOLE) = struct
> >
> > -  let start console = OS.Time.sleep 2.0 >>
> > +  let start console =
> > +    OS.Time.sleep 2.0
> > +
>
> Most people that have blank lines in their functions either don't know how
> to desconstruct large functions into named, compact and readable
> computations units (i.e. don't know how to write readable code) or have a
> style that is too imperative. Note that other old-timers will also tell you
> so see e.g. this message
> <https://sympa.inria.fr/sympa/arc/caml-list/2015-06/msg00143.html> by
> @mshinwell <https://github.com/mshinwell>.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/mattgray/devwinter2016/pull/2/files/edef802897029253c33be2490a719550ff1eee9b#r58861091>
>